src/main/java/io/stargate/sgv2/jsonapi/exception/ErrorCode.java

```diff
@@ -157,6 +157,8 @@ public enum ErrorCode {
   UNAUTHENTICATED_REQUEST("UNAUTHENTICATED: Invalid token"),
   COLLECTION_CREATION_ERROR(
       "Collection creation failure (unable to create table). Recommend re-creating the collection"),
+  OFFLINE_WRITER_SESSION_NOT_FOUND("Offline writer session not found :"),
```

Comment:
No need for trailing ` :` -- when using `ErrorCode.xxx.toApiException()` separate gets added automatically.

Reply:
Fixed in this commit [40402ef](https://github.com/stargate/data-api/pull/905/commits/40402efce668cca5834c2acc9fbc65ebebe1a8aa)